src/mumble/mumble_pch.hpp

```diff
@@ -121,8 +113,6 @@ typedef WId HWND;
 #include <mmintrin.h>
 #endif
 
-#define iroundf(x) ( static_cast<int>(x) )
```

Comment:
Shouldn’t this file include the new `Utils.h` then? If not, the commit should describe why this is not necessary nor desired.

Reply:
Done.